src/custom/pages/Profile/index.tsx

```diff
@@ -174,3 +175,7 @@ const formatDecimal = (number?: number): string => {
 const formatInt = (number?: number): string => {
   return number ? number.toLocaleString() : '-'
 }
+
+const StyledNotificationBanner = styled(NotificationBanner)`
```

Comment:
Looks great!

Minor detail: Perhaps we should move the Styled component below the imports.